contrib/guix/manifest.scm

```diff
@@ -257,9 +256,6 @@ and abstract ELF, PE and MachO formats.")
     (build-system cmake-build-system)
     (inputs
      `(("openssl", openssl)))
-    (arguments
-     '(#:configure-flags
-        (list "-DCMAKE_DISABLE_FIND_PACKAGE_CURL=TRUE")))
```

Comment:
641897a83dc9d40b618efbae67c3beb90a1f34f8: What did this do?

Reply:
Please see https://github.com/bitcoin/bitcoin/pull/27779.

Reply:
> What did this do?

Effectively, [nothing](https://github.com/bitcoin/bitcoin/pull/27779#issuecomment-1572754794).